src/module.c

```diff
@@ -5964,15 +5966,15 @@ int RM_DictDel(RedisModuleDict *d, RedisModuleString *key, void *oldval) {
  * comparison operator to use in order to seek the first element. The
  * operators available are:
  *
- * "^"   -- Seek the first (lexicographically smaller) key.
- * "$"   -- Seek the last  (lexicographically biffer) key.
- * ">"   -- Seek the first element greater than the specified key.
- * ">="  -- Seek the first element greater or equal than the specified key.
- * "<"   -- Seek the first element smaller than the specified key.
- * "<="  -- Seek the first element smaller or equal than the specified key.
- * "=="  -- Seek the first element matching exactly the specified key.
+ * * "\^"  -- Seek the first (lexicographically smaller) key.
```

Comment:
maybe we can somehow fix the ruby code to handle that instead of messing up the C comment?
(if we do that, we need to take code of the second reference to `^` below.

Reply:
I'll use backticks here.

I think it would be nice to stick to standard markdown as much as possible and not do too much processing in the script. Otherwise, we get a pretty complex semi-markdown language.

Reply:
yeah. you're right.
so let's try to use backtick instead of escaping. it'll look better when reading the source code.